public/app/plugins/panel/geomap/layers/data/MarkersLegend.tsx

```diff
@@ -50,33 +51,17 @@ export function MarkersLegend(props: MarkersLegendProps) {
     return <div className={style.infoWrap}>no thresholds????</div>;
   }
 
+  const items = getThresholdItems(color.field!.config, config.theme2);
```

Comment:
kinda unrelated.... but I have too many open PRs already!

This reuses the `getThresholdItems` function rather than an almost identical implementation